pam/integration-tests/vhs-helpers_test.go

```diff
@@ -262,6 +262,14 @@ func (td tapeData) RunVhs(t *testing.T, testType vhsTestType, outDir string, cli
 		"HOME="+os.Getenv("HOME"),
 	)
 
+	// Force VHS to use color because it makes it a lot easier to find the error
+	// message in the VHS output. The only case where we don't want color is when
+	// building a Debian package, because when viewing the logs of a launchpad build
+	// in the browser, ANSI colors are not rendered.
+	if os.Getenv("DEB_BUILD_ARCH") == "" {
```

Comment:
Wasn't the previous check on terminal working?

However, here we can just rely on an env variable that we set in `debian/rules` instead of guessing on something that may be set by debuild. So `AUTHD_TESTS_DISABLE_COLORS` or something.

Reply:
I used `DEB_BUILD_ARCH` because it's already used in the PAM integration tests: https://github.com/ubuntu/authd/blob/8b9dadb8607ded4b1f0c347593653531f8938b2c/pam/integration-tests/modulehelpers_test.go#L61-L61

maybe I misunderstood that usage?

Reply:
@3v1n0 do you know if there is a reliable way to detect if the tests are run on a launchpad builder or as part of a debian package build? I also need that for https://github.com/ubuntu/authd/pull/1087.